backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -1500,13 +1518,15 @@ def bind(self, configurator: Configurator) -> None:
         # set content as read/unread
         configurator.add_route(
             "read_content",
-            "/users/{user_id:\d+}/workspaces/{workspace_id}/contents/{content_id}/read",  # noqa: W605
+            "/users/{user_id:\d+}/workspaces/{workspace_id}/contents/{content_id}/read",
+            # noqa: W605
             request_method="PUT",
         )
         configurator.add_view(self.set_content_as_read, route_name="read_content")
         configurator.add_route(
             "unread_content",
-            "/users/{user_id:\d+}/workspaces/{workspace_id}/contents/{content_id}/unread",  # noqa: W605
+            "/users/{user_id:\d+}/workspaces/{workspace_id}/contents/{content_id}/unread",
+            # noqa: W605
```

Comment:
pourquoi y-a-t-il ce changement @Millefeuille42 ?

Reply:
It has been done when running the lint tools. Not sure of *why* these changes have been made by it though.

Reply:
resolved